wire/internal/wire/wire.go

```diff
@@ -234,7 +234,8 @@ func (g *gen) inject(pos token.Pos, name string, sig *types.Signature, set *Prov
 			}
 			if g.values[c.valueExpr] == "" {
 				t := c.valueTypeInfo.TypeOf(c.valueExpr)
-				name := disambiguate("_wire"+export(typeVariableName(t))+"Value", g.nameInFileScope)
+
+				name := typeVariableName(t, "", func(name string) string { return "_wire" + export(name) + "Value" }, g.nameInFileScope)
```

Comment:
It was surprising to me that this default is the empty string. I realize root cause and remarked below.